demo/index.html

```diff
@@ -5,12 +5,15 @@
     <title>SQLite Wasm Demo</title>
     <script type="module" src="script.js"></script>
     <script type="module" src="main-thread.js"></script>
+    <script type="module" src="sqlite.client.js"></script>
```

Comment:
Can you stick to the `-` naming convention?

Reply:
Done.